tomviz/ModuleContour.cxx

```diff
@@ -410,6 +430,8 @@ std::string ModuleContour::getStringForProxy(vtkSMProxy* proxy)
     return "Representation";
   } else if (proxy == this->ResampleFilter.Get()) {
     return "Resample";
+  } else if (proxy == this->PointDataToCellDataFilter.Get()) {
+    return "Resample";
```

Comment:
Yes, this is for saving animations.  I need to figure out which proxy is being animated and restore the animation to the right one.  ParaView's mechanism using proxy ids can't work here since we don't/can't save all of the pieces needed to make it work... so this was my workaround.  `ModuleX::getStringForProxy` and `ModuleX::getProxyForString` should be inverses to make this save/load logic work.